api/README.md

```diff
@@ -21,14 +21,56 @@ __Parameters__
 
 __`sha`__ : SHA[0:10] of the runs to get, or the keyword `latest`. Defaults to `latest`.
 
+__`product`__ : Product to include (repeated param), e.g. `chrome` or `firefox-60`
```

Comment:
`product` appears to be repeatable here and elsewhere, but this isn't indicated.

Reply:
It's repeatable for `runs` but not for a single `run`.